tailscale/client.go

```diff
@@ -1096,6 +1096,66 @@ func (c *Client) RotateWebhookSecret(ctx context.Context, endpointID string) (*W
 	return &webhook, c.performRequest(req, &webhook)
 }
 
+const (
+	ContactAccount  ContactType = "account"
+	ContactSupport  ContactType = "support"
+	ContactSecurity ContactType = "security"
+)
+
+type (
+	// ContactType defines the type of contact.
+	ContactType string
+
+	// Contacts type defines the object returned when retrieving contacts.
+	Contacts struct {
+		Account  Contact `json:"account"`
+		Support  Contact `json:"support"`
+		Security Contact `json:"security"`
+	}
+
+	// Contact type defines the structure of an individual contact for the tailnet.
+	Contact struct {
+		Email string `json:"email"`
+		// FallbackEmail is the email used when Email has not been verified.
+		FallbackEmail string `json:"fallbackEmail,omitempty"`
+		// NeedsVerification is true if Email needs to be verified.
+		NeedsVerification bool `json:"needsVerification"`
+	}
+
+	// UpdateContactRequest type defines the structure of a request to update a Contact.
+	UpdateContactRequest struct {
```

Comment:
I think this is the right shape, but I'm not sure if we want to end up with a bunch of types called `XXXRequest`. That said, I can't think of a better option.